docs-devsite/firestore_.memorycachesettings.md

```diff
@@ -1,35 +0,0 @@
-Project: /docs/reference/js/_project.yaml
```

Comment:
@hsubox76 I can't figure out why there are changes to firestore docs in my branch. Do you have an idea why?